<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_Estoque</class>
 <widget class="QMainWindow" name="main_Estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Estoque</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="bottomMargin">
     <number>35</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="botao_inicio">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INICIO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botao_cadastrar">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CADASTRAR PRODUTO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botao_listar">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LISTAR PRODUTOS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="Pages">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_listar">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>-2</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Poppins</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>EXCLUIR PRODUTO</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>ID: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_id_excluir"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="botao_excluir">
             <property name="font">
              <font>
               <family>Poppins</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>EXCLUIR</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_cadastrar">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CADASTRO DE PRODUTOS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nome</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nome_P"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Quantidade</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_qnt_P"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Data de Validade</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text_data_P">
            <property name="inputMask">
             <string>xx/xx/xxxx</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pre√ßo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_preco_P">
            <property name="inputMask">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botao_cadProduto">
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ENVIAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_inicio">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Poppins</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(220, 220, 220);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;ESTOQUE&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Controle de Estoque&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_cadUsuario">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CADASTRO DE USU√ÅRIOS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nome:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nome"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Usu√°rio:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_usuario"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Senha:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senha">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="rightMargin">
           <number>300</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Confirmar senha:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_confirmarsenha">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botao_cadastrar_2">
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>CADASTRAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botao_cadastrarUsu">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CADASTRAR USU√ÅRIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
